src/sentry/integrations/jira/integration.py

```diff
@@ -765,7 +780,7 @@ def create_issue(self, data, **kwargs):
         try:
             response = client.create_issue(cleaned_data)
         except Exception as e:
-            self.raise_error(e)
+            return self.raise_error(e)
```

Comment:
@mgaeta are there cases when `raise_error` doesn't actually raise ann error?

Reply:
It always `raise`s or calls `six.reraise`.